src/app/domain/backup/services/uploadMedia.ios.ts

```diff
@@ -51,7 +51,6 @@ export const uploadMedia = async (
 
   return uploadFileWithConflictStrategy({
     url: uploadUrl,
-    // @ts-expect-error Type issue which will be fixed in another PR
     token: client.getStackClient().token.accessToken as string,
```

Comment:
as I'm not sure of the actual interface in runtime, my typings only assume the key can exist, but could be undefined. I'm adding a runtime check here too, then. Throwing an unknown error has it seems an unlikely scenario to me

Reply:
Did not know about the '!' in TS. I added these because I found differences between types and actual content of client.getStackClient().token. I need to do a pass of cozy-client backup related types on flagship app...